### PR TITLE
updated details

### PR DESCRIPTION
**Pull request by Yogeshwari-Rathod**
please mrege changes file1

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
 write below details
-name -
-email id
-mob number
+name - sharlin kapadiya
```
Comment (sandeephoke) [inline file1]: please make it capital

**Review by sandeephoke — CHANGES_REQUESTED** [review]
please do the changes mentioned in the comment

File: file1 (hunk 1 above)
Comment (Yogeshwari-Rathod) [reply file1]: code changed done

Comment (Yogeshwari-Rathod) [reply file1]: please review and approve pull request

**Pull request merged**